src/containers/app/ScrollToTop.js

```diff
@@ -0,0 +1,20 @@
+/*
+ * @flow
+ */
+
+import React from 'react';
+import {withRouter} from "react-router-dom";
+
+class ScrollToTop extends React.Component {
+  componentDidUpdate(prevProps) {
+    if (this.props.location !== prevProps.location) {
```

Comment:
I believe `location` is an object, so this comparison won't work as you intended.